config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
 
+  devise_for :admin_users, ActiveAdmin::Devise.config
```

Comment:
Could we add this further down the routes file? Just to keep everything organised. You might do a clean up of the routes file when are you on it. Like for example the rate route should be further down.